lib/reporters/console_reporter.js

```diff
@@ -202,6 +202,21 @@ function ConsoleReporter() {
       printNewline();
       print(indent(stackFilter(failedExpectation.stack), 4));
     }
+    
+    // HACK: When failSpecWithNoExpectations = true and a spec fails because of no expectations found,
```

Comment:
If you have ideas for improving the reporter interface, I'm happy to have a discussion about that or to review pull requests. But let's keep this kind of editorializing out of the code.

Reply:
I tweaked the comment to get rid of the hack part - although it doesn't make it less true :). Spreading this piece of knowledge in the reporters feels like a compromise; probably due to the fact that adding a dedicated status would have been even more of a breaking change for reporters.